### ISSUE TITLE
Piechart color rendering problem

### ISSUE THREAD

**Issue by vicmns** [PhilJay/MPAndroidChart#2754]
Hey guys, first thanks for your awesome work!

I'm having a little rendering problem with my dataset. As you can see on the Image the pie chart is rendering a bunch of my data with the same color
![screenshot_1485368953](https://cloud.githubusercontent.com/assets/1101060/22303847/b2a5baae-e2fa-11e6-9b0c-b03bfb1d6c5e.png)
But once I select a data slice, this one is rendered with the correct color.
![screenshot_1485369048](https://cloud.githubusercontent.com/assets/1101060/22303854/ba5a1a9c-e2fa-11e6-8275-8dc0c651295a.png)
My datase is about 194 data slices.
I also modified the slice space and selection shift like this:
```
dataSet.setSliceSpace(colEdge.getEx() > 100 ? 0f : 3f);
dataSet.setSelectionShift(colEdge.getEx() > 100 ? 1f : 5f);
```

Do you guys have any idea on how to prevent this... I tried to reproduce it on the sample app with no luck 😞

**Comment by vicmns** [comment]
And here's a little gif of the issue
![render_problem](https://cloud.githubusercontent.com/assets/1101060/22304343/a10c51fc-e2fc-11e6-8c28-676781a66a15.gif)

**Comment by vicmns** [comment]
My bad, this issue was already fixed on Version 3.0.1... I have a mess on my current fork so I didn't know which version I had merge on... 

Closing this issue
